blog/2016-03_mongoc-uri.md

```diff
@@ -1,30 +1,19 @@
-# MongoDB destination receiving a face lift
```

Comment:
Thanks for the improvements. I've integrated some of the improvements in https://github.com/bkil-syslogng/syslog-ng-notes/commit/a77afa1611619cda326a11788ee114c2143db123 . Please rebase to master or open a new pull request if you have further additions.